EnergyCorrelatorOptions.h

```diff
@@ -166,6 +172,8 @@ namespace EnergyCorrelatorOptions {
       .ptJetBins     = ptJetBins,
       .jetAccept     = GetJetAccept(),
       .cstAccept     = GetCstAccept(),
+      .manualTwoPoint = manualTwoPoint,
+      .manualThreePoint = manualThreePoint,
```

Comment:
For consistency: I would recommend putting this option at the same point in the list. (Helps make the code more digestible.)

```suggestion
      .ptJetBins     = ptJetBins,
      .manualTwoPoint = manualTwoPoint,
      .manualThreePoint = manualThreePoint,
      .jetAccept     = GetJetAccept(),
      .cstAccept     = GetCstAccept()
```